graphql/shop/resolver.ts

```diff
@@ -50,6 +50,18 @@ const shopResolver = {
       });
     },
     nearByShops: async (parent, args: QueryNearByShopsArgs, ctx: Context) => {
+      const pendingTurns = await ctx.prisma.issuedNumber.findMany({
+        where: {
+          clientId: ctx.tokenInfo?.clientId,
+          status: 0,
+        },
+      });
+
+      let notIn = '';
+      pendingTurns.forEach((t) => {
+        notIn = !!notIn ? `${notIn},"${t.shopId}"` : `"${t.shopId}"`;
+      });
+
```

Comment:
I'm not sure about filtering out shops with turns here:
1. it's adding another costly db round trip without adding value
2. the pendingTurns query is running on each paginated request, whereas the pending turns remain the same
3. the client may not understand why the shop was removed from the list

I believe we should show at most a status indicator in the Request Turn page so the user knows they already have a number. In the meantime, I added a catch in the Request Turn button to automatically redirect the user to the home screen no matter if they have a number or not, I think that if they want to press that button again, we should let them.